exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity === studentCount + mentorCount; // finish this statement
```

Comment:
Great work Oksana. I only suggest this part. Because  yes it can return as true, but it may not be working properly. Please consider below alternatives. Well done.
var enoughSpaceInRoom = roomMaxCapacity >= studentCount + mentorCount;
var enoughSpaceInRoom = roomMaxCapacity >= 25;